package.json

```diff
@@ -79,6 +79,7 @@
       "tslib": "^2.5.0",
       "ts-node": "^10.9.1",
       "typescript": "^5.0.3",
+      "xlsx": "https://cdn.sheetjs.com/xlsx-0.19.3/xlsx-0.19.3.tgz",
```

Comment:
[sheetsjs has stopped publishing to NPM for over a year ago](https://git.sheetjs.com/sheetjs/sheetjs/issues/2667), and this is the only way to get updates.

Reply:
Interesting stuff "Due to ongoing legal matters between SheetJS LLC and npm" 🤨